Mathlib/Algebra/Homology/DerivedCategory/ExactFunctor.lean

```diff
@@ -91,7 +92,26 @@ instance : NatTrans.CommShift F.mapDerivedCategoryFactors.hom ℤ :=
       simp only [id_comp, mapDerivedCategoryFactorsh_hom_app, assoc, comp_id,
         ← Functor.map_comp_assoc, Iso.inv_hom_id_app, map_id, comp_obj])
 
+instance : NatTrans.CommShift F.mapDerivedCategoryFactors.symm.hom ℤ :=
+  NatTrans.CommShift.of_iso_inv F.mapDerivedCategoryFactors ℤ
+
 instance : F.mapDerivedCategory.IsTriangulated :=
   Functor.isTriangulated_of_precomp_iso F.mapDerivedCategoryFactorsh
 
+/-- The commute of `DerivedCategory.singleFunctor` with `F` and `F.mapDerivedCategory`. -/
```

Comment:
```suggestion
/-- `DerivedCategory.singleFunctor` commutes with `F` and `F.mapDerivedCategory`. -/
```
Commute is not a noun. This is one fix but there might be better ones. Same for the other docstrings.